src/ucp/core/ucp_ep.c

```diff
@@ -1801,3 +1821,17 @@ ucp_wireup_ep_t * ucp_ep_get_cm_wireup_ep(ucp_ep_h ep)
     return ucp_wireup_ep_test(ep->uct_eps[lane]) ?
            ucs_derived_of(ep->uct_eps[lane], ucp_wireup_ep_t) : NULL;
 }
+
+uct_ep_h ucp_ep_get_cm_uct_ep(ucp_ep_h ep)
+{
+    ucp_lane_index_t lane;
+    ucp_wireup_ep_t *wireup_ep;
+
+    lane = ucp_ep_get_cm_lane(ep);
+    if (lane == UCP_NULL_LANE) {
+        return NULL;
+    }
+
+    wireup_ep = ucp_ep_get_cm_wireup_ep(ep);
```

Comment:
how can CM be part of wireup ep?

Reply:
see https://github.com/openucx/ucx/pull/4417/files#r346803351